core/templates/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.service.ts

```diff
@@ -18,16 +18,21 @@
 require('components/ck-editor-helpers/ck-editor-4-rte.directive.ts');
 require('components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts');
 
-require('domain/utilities/url-interpolation.service.ts');
-require('services/suggestion-modal.service.ts');
 require(
   'pages/creator-dashboard-page/suggestion-modal-for-creator-view/' +
-  'suggestion-modal-for-creator-view.controller');
+'suggestion-modal-for-creator-dashboard-backend-api.service.ts');
+require(
+  'pages/creator-dashboard-page/suggestion-modal-for-creator-view/' +
+'suggestion-modal-for-creator-view.controller');
```

Comment:
Indent by two spaces.
```suggestion
  'suggestion-modal-for-creator-view.controller');
```

Reply:
Done.